### PR TITLE
Feature/ex 4242 update archetype documentation a

### PR DESCRIPTION
**Pull request by davidmfempathy**
# What

- Update Archetype documentation according to the new Github scenario. I've include information about the DS, how to clone and run it, etc.
- Update i18n documentation according to the new x-translations package added recently.

# Motivation

The documentation was outdated.

# Test

- Check that the milestones from the task are fulfilled.
- Suggest any new change.

### CODE REVIEW

**Review by tiborux — DISMISSED** [review]

File: docs/index.md [hunk 1]
```diff
@@ -1,65 +1,60 @@
...
 
-The first step to start a new setup with the archetype is to clone the archetype project.
+The first step to start a new setup with the archetype is cloning the project.
 
-1. Create the new repository in bitbucket accessing this URL and entering the name of the new
-repository. The repository name `<new-repository-name>` will be used in the next commands:
-    
-    [https://bitbucket.org/repo/create?owner=colbenson](https://bitbucket.org/repo/create?owner=colbenson)  
-    ![create bitbucket repository](index-1-create-repository.png "create bitbucket repository")
+We recommend to use [Degit](https://github.com/Rich-Harris/degit) which let us copying a project without the `.git`
```
Comment (CachedaCodes) [inline docs/index.md:18]: Same as above about the file extension format

Comment (CachedaCodes) [inline docs/index.md:18]: Change to "We recommend using [Degit]".

- [ ] Fix typo

File: docs/index.md [hunk 2]
```diff
@@ -77,15 +72,51 @@ replaced for the repository name:
...
-file. The adapter is mandatory and is imported from the previous file. To know more about the
-plugin options visit: <!-- TODO: Add here the link to the Plugin doc page-->
+file. The adapter is mandatory, and it is imported from the previous file.
+
+### Styling with the design system
+
+X Components implements its own design system with *design components* and *design tokens*. You can easily modify the
+resultant application appearance overriding them in the predefined [tokens file](../src/design-system/tokens.scss).
+
+You can take a look to the [X Components design system README](https://github.com/empathyco/x/tree/main/packages/x-components/contributing/design-system.md)
```
Comment (CachedaCodes) [inline docs/index.md:89]: Change to "You can take a look at the".

- [ ] Fix typo

**Review by CachedaCodes — CHANGES_REQUESTED** [review]

File: docs/i18n.md [hunk 3]
```diff
@@ -149,3 +149,30 @@ initial bundle size. It will only be downloaded when it is needed.
 Because changing a locale may imply downloading new messages, a `setI18nConfiguration` method has been created
 as a helper. This method is exposed after initializing the internationalization with
 `createI18NInstance` in the [src/main.ts](../src/main.ts) file.
+
+### Managing i18n translations
+
+We created a library for managing the i18n translations easier. It transforms the application i18n files in .csv files
```
[the remark this answers is not in the document]
Comment (davidmfempathy) [reply docs/i18n.md:155]: I've changed both to .csv which makes clearer that it's an extension in my opinion.

**Review by CachedaCodes — APPROVED** [review]

**Review by tiborux — APPROVED** [review]

**Pull request merged**